php/class-plugin.php

```diff
@@ -13,38 +13,38 @@
  * Class Plugin
  */
 class Plugin extends Plugin_Abstract {
+
 	/**
-	 * WP Admin resources.
+	 * Utility methods.
 	 *
-	 * @var Admin\Admin
+	 * @var Util
 	 */
-	public $admin;
+	protected $util;
 
 	/**
-	 * Block Post Type.
+	 * WP Admin resources.
 	 *
-	 * @var Post_Types\Block_Post
+	 * @var Admin\Admin
 	 */
-	public $block_post;
+	public $admin;
 
 	/**
-	 * Initiate the loading of new blocks.
+	 * The slug of the post type that stores the blocks.
 	 *
-	 * @var Blocks\Loader
+	 * @since 1.3.5
+	 * @var string
 	 */
-	public $loader;
+	public $post_type_slug = 'block_lab';
```

Comment:
What do you think about just calling this `$slug` instead of `$post_type_slug` – purely for brevity?

Reply:
Yeah, that would be a little simpler. The reasoning was to distinguish it from the `$slug` property of `Plugin_Abstract`. 

Though both of them end up being `'block_lab'`

Reply:
Aah! Good point. Where is that used?

Reply:
Might be better to just go with the `Plugin_Abstract->$slug` property. The post type can just use that too.

Reply:
Though maybe `Util::get_post_type_slug()` could simply return `Plugin::$slug`, which also happens to be `'block_lab'`

Reply:
Ah, my fault. `Plugin::$slug` is `'block-lab'`, while `Plugin::$post_type_slug` is `'block_lab'`